test/uk/gov/hmrc/minorentityidentificationfrontend/utils/MessageHelperSpec.scala

```diff
@@ -35,7 +35,7 @@ class MessageHelperSpec extends AnyFlatSpec with BeforeAndAfterEach {
     val welshServiceName = "This is a welsh service name"
     val englishServiceName = "This is an english service name"
 
-    val welshLabels: JourneyLabels = JourneyLabels(welshServiceName)
+    val welshLabels: JourneyLabels = JourneyLabels(Some(welshServiceName), Some(englishServiceName))
```

Comment:
journeyLabels would be better name, as labels now can be both welsh and english

Reply:
done